.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ commands:
 jobs:
   install-operator:
     docker:
-      - image: cimg/go:1.18
```

Comment:
Is Golang 1.20 ready to be used in the productization pipeline?

Reply:
hmm good question , I'm not sure , if not we will have to put this on hold for a while.